apps/labs/posts/numba-polynomial-support.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Adding support for polynomials to Numba'
+authors: [kristian-minchev]
+published: September 18, 2023
+description: 'My work was focused on improving NumPy support in Numba, with focus on the polynomial package.'
+category: [PyData ecosystem]
+featuredImage:
+  src: /posts/numba-polynomial-support/feature_grayscale.png
+  alt: >
+    An image containing photos of Kristian Minchev, intern at Quansight Labs, and Guilherme Leobas, Kristian's mentor during the internship.
+hero:
+  imageSrc: /posts/numba-polynomial-support/hero.png
+  imageAlt: 'An image containing the logos of Numba and NumPy next to each other.'
+---
+
+Hi, my name is Kristian Minchev. In this blog post, I will be talking about my experience as a summer intern at Quansight Labs working on enhancing NumPy support in Numba. In particular, my work was focused on adding support for the Polynomial class as well as other functions from the polynomial API.
+
+<p float="right">
+  <img alt="Numba logo."
+       src="/posts/numba-polynomial-support/numba_logo.png" width="45%" />
+  <img alt="Quansight Labs logo."
+       src="/posts/numba-polynomial-support/qlabs_logo.png"
+       width="45%" /> 
+</p>
```

Comment:
A question about this, @KrisMinchev: were you trying to get the Numba and Labs logos to float to the right of the text introducing yourself (starting on line 16)? Else, please explain what you'd like this to look like.

Either way, I should be able to get it set up how you were imagining.